config/_shared.json

```diff
@@ -18,6 +18,7 @@
 	"google_analytics_enabled": null,
 	"google_analytics_key": null,
 	"google_adwords_conversion_id": null,
+	"lucky_orange_enabled": null,
```

Comment:
Two things:
- yes, it should be a Boolean
- I'm not sure this is the place for these flags. I understand we are not talking about tangible user-facing features, but these analytics flags are probably still feature flags, if nothing else for architectural soundness. @dmsnell may have comments since that Directly episode.